packages/json-serializer/test/serializeJSON-test.js

```diff
@@ -22,16 +22,18 @@ describe('#serializeJSON', () => {
     const element = new namespace.elements.Element();
     element.element = 'message';
 
+    const dataStructure = new namespace.elements.Element();
+    dataStructure.element = 'dataStructure';
```

Comment:
```suggestion
    const dataStructure = new namespace.elements.DataStructure();
```

There’s a DataStructure element which can be used, that way line 26 won’t be needed (https://apielements.org/projects/js/en/latest/api.html#DataStructure).